lms/templates/discussion/_thread_list_template.html

```diff
@@ -1,17 +1,6 @@
 <%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 <script type="text/template" id="thread-list-template">
-    <div class="forum-nav-header">
-        <button type="button" class="forum-nav-browse" id="forum-nav-browse" aria-haspopup="true">
```

Comment:
I don't love this UI either, but I think it's better than all the other alternatives we can accomplish using the time frame and the resources we've been given.